src/e2e/java/teammates/e2e/cases/lnp/BaseLNPTestCase.java

```diff
@@ -234,4 +234,11 @@ protected void deleteDataFiles() throws IOException {
         Files.delete(Paths.get(pathToCsvFile));
     }
 
+    /**
+     * Sanitize the JSON string to be CSV data.
+     */
+    protected String sanitizeJsonString(String json) {
+        return String.format("\"%s\"", json.replace("\n", "").replace("\"", "\"\""));
```

Comment:
- The method name should be `sanitizeForCsv` because (1) it needs not be JSON and (2) the sanitization policy is specifically for CSV
- You may want to use `System.lineSeparator()` instead of hard-coded `\n`